qemu/tests/virt_firmware_check_phys_bits.py

```diff
@@ -0,0 +1,98 @@
+import re
+from avocado.utils import process
+from virttest import virt_vm
+from virttest import env_process
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Test parameters host-phys-bits and host-phys-bits-limit.
+    host-phys-bits=on is default on rhel qemu-kvm builds,
+    so cover the default value in the test.
+    phys-bits to be covered in the test:
+    AMD processors have 40, 43 or 48 phys-bits.
+    Intel processors have 36, 39, 46 or 52 phys-bits.
+    In this case, the cpu will be configured with host-phys-bits=on
+    and host-phys-bits-limit=<phys-bits-to-test>.
+    The maximum number OVMF is using right now is 46.
+    For 5-level page inside OVMF, they don't have that yet.
```

Comment:
then do we need to set the max number in a variant, in case we support it(larger than 46) one day...

Reply:
Had set limitation_from_ovmf = 46, and I will update limitation_msg in cfg file. Thanks.